iothub_client/src/iothubtransport_mqtt_common.c

```diff
@@ -2583,6 +2585,12 @@ IOTHUB_CLIENT_RESULT IoTHubTransport_MQTT_Common_SetOption(TRANSPORT_LL_HANDLE h
             mqtt_client_set_trace(transport_data->mqttClient, transport_data->log_trace, transport_data->raw_trace);
             result = IOTHUB_CLIENT_OK;
         }
+        else if (strcmp(OPTION_SAS_TOKEN_LIFETIME, option) == 0)
+        {
+            size_t* sas_lifetime = (size_t*)value;
```

Comment:
>size_t* sas_lifetime =  [](start = 12, length = 23)

Why doing this?

Reply:
Github issue

Reply:
Just to be specific.

---
In reply to: [140592649](https://github.com/Azure/azure-iot-sdk-c/pull/240#discussion_r140592649) [](ancestors = 140592649)